src/main/webapp/app/admin/logs/logs.component.html

```diff
@@ -1,28 +1,35 @@
+    <table class="table table-sm table-striped table-bordered" aria-describedby="logs-page-heading">
         <thead>
-            <tr title="click to order">
-                <th (click)="orderProp = 'name'; reverse = !reverse"><span jhiTranslate="logs.table.name">Name</span></th>
-                <th (click)="orderProp = 'level'; reverse = !reverse"><span jhiTranslate="logs.table.level">Level</span></th>
+            <tr jhiSort [(predicate)]="orderProp" [(ascending)]="ascending" (sortChange)="filterAndSort()">
+                <th jhiSortBy="name" scope="col"><span jhiTranslate="logs.table.name">Name</span> <fa-icon icon="sort"></fa-icon></th>
+                <th jhiSortBy="level" scope="col"><span jhiTranslate="logs.table.level">Level</span> <fa-icon icon="sort"></fa-icon></th>
             </tr>
         </thead>
 
-        <tr *ngFor="let logger of loggers | pureFilter: filter:'name' | orderBy: orderProp:reverse">
+        <tr *ngFor="let logger of filteredAndOrderedLoggers">
             <td>
                 <small>{{ logger.name | slice: 0:140 }}</small>
             </td>
             <td>
-                <button (click)="changeLevel(logger.name, 'TRACE')" [ngClass]="logger.level == 'TRACE' ? 'btn-danger' : 'btn-light'" class="btn btn-sm">TRACE</button>
-                <button (click)="changeLevel(logger.name, 'DEBUG')" [ngClass]="logger.level == 'DEBUG' ? 'btn-warning' : 'btn-light'" class="btn btn-sm">DEBUG</button>
-                <button (click)="changeLevel(logger.name, 'INFO')" [ngClass]="logger.level == 'INFO' ? 'btn-info' : 'btn-light'" class="btn btn-sm">INFO</button>
-                <button (click)="changeLevel(logger.name, 'WARN')" [ngClass]="logger.level == 'WARN' ? 'btn-success' : 'btn-light'" class="btn btn-sm">WARN</button>
-                <button (click)="changeLevel(logger.name, 'ERROR')" [ngClass]="logger.level == 'ERROR' ? 'btn-primary' : 'btn-light'" class="btn btn-sm">ERROR</button>
+                <button (click)="changeLevel(logger.name, 'TRACE')" [ngClass]="logger.level === 'TRACE' ? 'btn-primary' : 'btn-light'" class="btn btn-sm">TRACE</button>
+
+                <button (click)="changeLevel(logger.name, 'DEBUG')" [ngClass]="logger.level === 'DEBUG' ? 'btn-success' : 'btn-light'" class="btn btn-sm">DEBUG</button>
+
+                <button (click)="changeLevel(logger.name, 'INFO')" [ngClass]="logger.level === 'INFO' ? 'btn-info' : 'btn-light'" class="btn btn-sm">INFO</button>
+
+                <button (click)="changeLevel(logger.name, 'WARN')" [ngClass]="logger.level === 'WARN' ? 'btn-warning' : 'btn-light'" class="btn btn-sm">WARN</button>
+
+                <button (click)="changeLevel(logger.name, 'ERROR')" [ngClass]="logger.level === 'ERROR' ? 'btn-danger' : 'btn-light'" class="btn btn-sm">ERROR</button>
+
+                <button (click)="changeLevel(logger.name, 'OFF')" [ngClass]="logger.level === 'OFF' ? 'btn-secondary' : 'btn-light'" class="btn btn-sm">OFF</button>
```

Comment:
Why adding the newlines? I don't think this increases readability. I suggest removing the newlines inbetween.

Reply:
Not necessary since generated from jhipster